dev/fattest.simplicity/src/componenttest/topology/utils/HttpUtils.java

```diff
@@ -61,6 +61,8 @@ public class HttpUtils {
 
     private final static int LOWEST_ERROR_CODE = 400;
 
+    public static final String PEER_CERTIFICATES = "PEER_CERTIFICATES";
```

Comment:
Not sure what the point of this change is, now stuff is back in GVT Utils.

Reply:
Removed the change